msf4j/pom.xml

```diff
@@ -0,0 +1,16 @@
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>msf4j</artifactId>
+    <packaging>pom</packaging>
+    <parent>
+        <artifactId>parent-modules</artifactId>
+        <groupId>com.baeldung</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <modules>
```

Comment:
Typically, it's preferred to differentiate examples by package instead of by maven module. Would you be able to condense this to one project, `msf4j`, and then separate the examples by package?